runtime/src/iree/hal/vulkan/vulkan_device.cc

```diff
@@ -620,11 +620,11 @@ static iree_status_t iree_hal_vulkan_device_create_internal(
 
   // Emulate timeline semaphores when the extension is not available and we are
   // ony Vulkan versions prior to 1.2 when they were made core.
-  bool emulate_timeline_semaphores =
-      device_syms->vkGetSemaphoreCounterValue == NULL ||
-      iree_all_bits_set(
-          options->flags,
-          IREE_HAL_VULKAN_DEVICE_FORCE_TIMELINE_SEMAPHORE_EMULATION);
+  bool emulate_timeline_semaphores = false;
```

Comment:
Want me to rebase https://github.com/google/iree/pull/8018 ? :)

Reply:
yeah! that'd be great!